### PR TITLE
feat(listbox): expose and inject background image

### PR DESCRIPTION
**Pull request by johanlahti**
## Motivation

This PR enables properties for Listbox background image in addition to color.

## Requirements checklist

- [x] Api specification
  - [x] Ran `yarn spec`
    - [x] No changes **_OR_** API changes has been formally approved
- [x] Unit/Component test coverage
- [x] Correct PR title for the changes (fix, chore, feat)

When build and tests have passed:

- [x] Add code reviewers, for example @qlik-oss/nebula-core

### CODE REVIEW

**Review by DanielS-Qlik — APPROVED** [review]

File: apis/nucleus/src/utils/style/styling-props.js [hunk 1]
```diff
@@ -51,8 +51,9 @@ function getBackgroundPosition(bgComp) {
 
 function getBackgroundSize(bgComp) {
   let bkgImageSize = imageSizingToCssProperty.originalSize;
-  if (bgComp?.bgImage?.sizing) {
-    bkgImageSize = imageSizingToCssProperty[bgComp.bgImage.sizing];
+  const size = bgComp?.bgImage?.sizing || bgComp?.bgImage?.size;
```
Comment (veinfors) [inline apis/nucleus/src/utils/style/styling-props.js:54]: It's a little annoying that the api differs here and that we have to support two cases like this.
Don't know if it's "too late" to change or we have good reason to add `size` as well.

File: apis/nucleus/src/utils/style/styling-props.js [hunk 2]
```diff
@@ -67,9 +68,9 @@ export function resolveBgImage(bgComp, app) {
   if (bgImageDef) {
     let url = '';
     if (bgImageDef.mode === 'media' || bgComp.useImage === 'media') {
-      url = bgImageDef?.mediaUrl?.qStaticContentUrl?.qUrl
-        ? decodeURIComponent(bgImageDef.mediaUrl.qStaticContentUrl.qUrl)
-        : undefined;
+      const urlObj = bgImageDef?.mediaUrl || bgImageDef?.url;
```
Comment (veinfors) [inline apis/nucleus/src/utils/style/styling-props.js:71]: Same as the comment above but for `url` here. Any specific reason to why we add yet another prop for the same purpose.

File: apis/nucleus/src/utils/style/styling-props.js (hunk 1 above)
Comment (johanlahti) [reply apis/nucleus/src/utils/style/styling-props.js:54]: Yeah, I acquired this from another place (Action button) which used these keys.

Comment (johanlahti) [reply apis/nucleus/src/utils/style/styling-props.js:54]: It's not too late to change. I can change to what Nebula used before.

**Pull request merged**